src/smart-components/access-management/UsersTable.tsx

```diff
@@ -72,18 +79,31 @@ const UsersTable: React.FunctionComponent = () => {
     }
   };
 
-  const rows = users.map((user: UserProps) => [
-    user.username,
-    user.email,
-    user.first_name,
-    user.last_name,
-    user.is_active ? 'Active' : 'Inactive',
-    user.is_org_admin ? 'Yes' : 'No',
-    {
-      cell: <ActionsColumn items={ROW_ACTIONS} />,
-      props: { isActionCell: true },
-    },
-  ]);
+  const rows = useMemo(() => {
+    return users.map((user: any) => [
+      user.username,
+      user.email,
+      user.first_name,
+      user.last_name,
+      user.is_active ? 'Active' : 'Inactive',
+      user.is_org_admin ? 'Yes' : 'No',
+      {
+        cell: (
+          <ActionsColumn
+            items={[
+              { title: 'Add to user group', onClick: () => console.log('ADD TO USER GROUP') },
+              {
+                title: 'Remove from user groups',
+                onClick: (event: KeyboardEvent | React.MouseEvent, rowId: number, rowData: any) => handleModalToggle(event, rowData),
+              },
+            ]}
+            rowData={user}
+          />
+        ),
+        props: { isActionCell: true },
+      },
+    ]);
+  }, [users]);
```

Comment:
```suggestion
  const rows = useMemo(() => users.map((user: any) => [
      user.username,
      user.email,
      user.first_name,
      user.last_name,
      user.is_active ? 'Active' : 'Inactive',
      user.is_org_admin ? 'Yes' : 'No',
      {
        cell: (
          <ActionsColumn
            items={[
              { title: 'Add to user group', onClick: () => console.log('ADD TO USER GROUP') },
              {
                title: 'Remove from user groups',
                onClick: (event: KeyboardEvent | React.MouseEvent, rowId: number, rowData: any) => handleModalToggle(event, rowData),
              },
            ]}
            rowData={user}
          />
        ),
        props: { isActionCell: true },
      },
    ]), [users]);
```
Thank you for the changes, one last comment.
Why do we get rid of the specific UserProps type? We should not use any

Reply:
I just tried it with `User` and it seems to work 🤔 
```
  const rows = useMemo(
    () =>
      users.map((user: User) => [
        user.username,
        user.email,
        user.first_name,
        user.last_name,
        user.is_active ? 'Active' : 'Inactive',
        user.is_org_admin ? 'Yes' : 'No',
        {
          cell: (
            <ActionsColumn
              items={[
                { title: 'Add to user group', onClick: () => console.log('ADD TO USER GROUP') },
                {
                  title: 'Remove from user groups',
                  onClick: (event: KeyboardEvent | React.MouseEvent, rowId: number, rowData: any) => handleModalToggle(event, rowData),
                },
              ]}
              rowData={user}
            />
          ),
          props: { isActionCell: true },
        },
      ]),
    [users]
  );
```

Reply:
:man_facepalming: that's crazy. I must've goofed up somewhere when I tried it